external/lfric_infrastructure/update.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/bash
+
+# This script takes a version of the lfric_core repository, copies
+# all the required source files into the PSyclone tree, and preprocesses
+# all files (into a separate directory, to avoid issues with file system
```

Comment:
I suspect I'm missing something critical here but why do we have the original files *and* the pre-processed files in the repository?